src/WalletNavigator.tsx

```diff
@@ -140,6 +159,7 @@ const useStrings = () => {
     receiveTabBarLabel: intl.formatMessage(messages.receiveButton),
     delegateTabBarLabel: intl.formatMessage(messages.delegateButton),
     walletTabBarLabel: intl.formatMessage(messages.walletButton),
+    nftsTabBarLabel: (qty) => intl.formatMessage(messages.nftsButton, {qty}),
```

Comment:
I prefer to not have the number in the tab, it will be displayed on top of the gallery